gammapy/datasets/tests/test_map.py

```diff
@@ -276,7 +277,7 @@ def test_fake(sky_model, geom, geom_etrue):
 
     assert real_dataset.counts.data.shape == dataset.counts.data.shape
     assert_allclose(real_dataset.counts.data.sum(), 9525.299054, rtol=1e-5)
-    assert_allclose(dataset.counts.data.sum(), 9711)
+    assert_allclose(dataset.counts.data.sum(), 9709)
```

Comment:
This PR should not change any test values. I think it is related to my comment above, that you start out with a single  PSF for all model components, instead of extracting the one at the components position.

Reply:
The tests changes are not related to the precomputation of the PSFKernel.
After I moved the modifications of the get_psf_kernel to a separate PR (https://github.com/gammapy/gammapy/pull/5314) the tests failed there and are all good in this one.
Actually they come from the changes in upsampling factor, and in the previous PR that modified this I also had to change few test values ([#4975](https://github.com/gammapy/gammapy/pull/4975/files)).